### PR TITLE
Support symlinks when saving files

### PR DESCRIPTION
**Pull request by schliflo**

### CODE REVIEW

File: File/src/File.php [hunk 1]
```diff
@@ -325,6 +325,10 @@ public function save($data = null)
         if ($data !== null) {
             $this->content($data);
         }
+        
+        if (is_link($this->filename)) {
+            $this->filename = realpath($this->filename);
+        }
 
         $filename = $this->filename;
```
Comment (mahagr) [inline File/src/File.php:333]: Can you move the fix after this line and just use `$filename`? Because of otherwise the URLs will not stay intact.

Comment (schliflo) [reply File/src/File.php:329]: sure. done.

**Comment by mahagr** [comment]
Cool, thanks

**Comment by schliflo** [comment]
@mahagr this has not been released so far. Would you mind releasing this patch along with the next Grav CMS RC? Thanks a bunch!

**Comment by mahagr** [comment]
You can manually use the unreleased version by changing the `composer.json` file, if needed.

Thanks for reminding me. (cc @rhukster)

**Pull request merged**